cel/src/objects.rs

```diff
@@ -752,26 +824,40 @@ impl Value {
                                 (Value::String(_), Value::Int(idx)) => {
                                     Err(ExecutionError::NoSuchKey(idx.to_string().into()))
                                 }
-                                (Value::Map(map), Value::String(property)) => map
-                                    .get(&property.into())
-                                    .cloned()
-                                    .unwrap_or(Value::Null)
-                                    .into(),
-                                (Value::Map(map), Value::Bool(property)) => map
-                                    .get(&property.into())
-                                    .cloned()
-                                    .unwrap_or(Value::Null)
-                                    .into(),
-                                (Value::Map(map), Value::Int(property)) => map
-                                    .get(&property.into())
-                                    .cloned()
-                                    .unwrap_or(Value::Null)
-                                    .into(),
-                                (Value::Map(map), Value::UInt(property)) => map
-                                    .get(&property.into())
-                                    .cloned()
-                                    .unwrap_or(Value::Null)
-                                    .into(),
```

Comment:
This is a change in behaviour, missing keys for `Value::Map` would `unwrap` to `Value::Null` instead of correctly returning a `NoSuchKey` error.